src/handlers/comment/handlers/wallet.ts

```diff
@@ -69,6 +69,10 @@ export const registerWallet = async (body: string) => {
   }
 
   if (address) {
+    if (address == "0x0000000000000000000000000000000000000000"){
```

Comment:
Avoid using the hardcoded `0x00..0` address. there should be a AddressZero constant in ethers library.